tests/integration-tests/tests/common/utils.py

```diff
@@ -90,11 +90,56 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         sys.exit(1)
 
 
+def retrieve_pcluster_ami_without_standard_naming(region, os, version="2.8.1", architecture="x86_64"):
+    try:
+        client = boto3.client("ec2", region_name=region)
+        ami_name = f"pcluster-without-standard-naming-{version}-{os}"
+        amis = client.describe_images(
+            Filters=[{"Name": "name", "Values": [ami_name]}, {"Name": "architecture", "Values": [architecture]}],
+        ).get("Images", [])
+        if amis:
+            # If the AMI is found, return the AMI
+            return amis[0]["ImageId"]
+        else:
+            # If no AMI is found, copy from an official AMI
+            official_ami_name = "aws-parallelcluster-{version}-{ami_name}".format(
+                version=version, ami_name=OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP.get(os).get("name")
+            )
+            official_amis = (
+                boto3.client("ec2", region_name=region)
+                .describe_images(
+                    Filters=[
+                        {"Name": "name", "Values": [official_ami_name]},
+                        {"Name": "architecture", "Values": [architecture]},
+                    ],
+                    Owners=OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP.get(os).get("owners"),
+                )
+                .get("Images", [])
+            )
+            return client.copy_image(
+                Description="This AMI is a copy from a official AMI but uses different naming. "
+                "It is used to bypass the name validation of pcluster version validation",
```

Comment:
Minor: I would mention that this AMI is used by `test_create_wrong_pcluster_version` here in description, and include why we need this AMI in `test_create_wrong_pcluster_version`

Reply:
I will try to do it within 255 characters